Domain.fs

```diff
@@ -1,19 +1,26 @@
 module Domain
-    type Player = {
-        FirstName:string
-        LastName:string
-        Team: string
-        Position: string
-        Rating:int 
-    }
 
-    type Team = {
-        Goalkeeper: Player
-        Defenders: Player[]
-        Midfielder: Player[]
-        Attacker: Player[]
-    }
+type Player =
```

Comment:
@vasily-kirichenko is the blank line under module and no indentation F# guidelines or personal preference?? I can see I've styled the record types wrong everywhere, that's good to know, thanks